python38-torch-cuda/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM nvidia/cuda:11.6.0-base-ubuntu20.04
```

Comment:
20.04 is not LTS and also not very recent :thinking:

Reply:
I'll have a look

Reply:
Seems Cuda 11.6 isn't available with ubuntu 22.04
There's Cuda 11.7, let's see if that works

Reply:
Seems pytorch doesn't have a Cuda 11.7 version yet, either we'll have to build an image from base (a bit hard with cuda) or leave as is @tomhobson

Reply:
leave as is, python is a fckin nightmare for support isn't it

Reply:
python should be fine, but cuda is worse